src/code/z_eff_blure.c

```diff
@@ -66,40 +68,977 @@ void func_800A81F0(EffectBlure* this, Vec3f* p1, Vec3f* p2) {
+        EffectBlure_UpdateFlags(elem);
+    }
+
+    this->elements[this->numElements - 1].flags &= ~3;
+    this->elements[this->numElements - 1].flags |= 2;
+
+    EffectBlure_SetupSmooth(this, gfxCtx);
+    SkinMatrix_SetTranslate(&spDC, this->elements[0].p2.x, this->elements[0].p2.y, this->elements[0].p2.z);
+    SkinMatrix_SetScale(&sp9C, 0.1f, 0.1f, 0.1f);
+    SkinMatrix_MtxFMtxFMult(&spDC, &sp9C, &sp5C);
+
+    mtx = SkinMatrix_MtxFToNewMtx(gfxCtx, &sp5C);
+    if (mtx == NULL) {
+        return;
+    }
+
+    gSPMatrix(POLY_XLU_DISP++, mtx, G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+
+    for (i = 0, elem = &this->elements[0]; elem < this->elements + this->numElements - 1; i++, elem++) {
+        if ((elem->state == 0) || ((elem + 1)->state == 0)) {
+            continue;
+        }
+        if ((((elem->flags & 3) == 0) && (((elem + 1)->flags & 3) == 0)) ||
+            (((elem->flags & 3) == 2) && (((elem + 1)->flags & 3) == 0)) ||
+            (((elem->flags & 3) == 0) && (((elem + 1)->flags & 3) == 2)) ||
+            (((elem->flags & 3) == 2) && (((elem + 1)->flags & 3) == 2))) {
+            EffectBlure_DrawElemNoInterpolation(this, elem, i, gfxCtx);
+        } else {
+            EffectBlure_DrawElemHermiteInterpolation(this, elem, i, gfxCtx);
+        }
+    }
+
+    CLOSE_DISPS(gfxCtx);
+}
+
+void EffectBlure_SetupSimple(GraphicsContext* gfxCtx, EffectBlure* this, Vtx* vtx) {
+    OPEN_DISPS(gfxCtx);
+
+    POLY_XLU_DISP = Gfx_CallSetupDL(POLY_XLU_DISP, 0x26);
+
+    CLOSE_DISPS(gfxCtx);
+}
+
+void EffectBlure_SetupSimpleAlt(GraphicsContext* gfxCtx, EffectBlure* this, Vtx* vtx) {
+    OPEN_DISPS(gfxCtx);
+
+    gDPPipeSync(POLY_XLU_DISP++);
+    POLY_XLU_DISP = Gfx_CallSetupDL(POLY_XLU_DISP, 0x26);
+
+    gDPSetCycleType(POLY_XLU_DISP++, G_CYC_2CYCLE);
+    gDPSetTextureLUT(POLY_XLU_DISP++, G_TT_NONE);
+    gSPTexture(POLY_XLU_DISP++, 0xFFFF, 0xFFFF, 0, G_TX_RENDERTILE, G_ON);
+
+    gDPLoadTextureBlock(POLY_XLU_DISP++, D_04014570, G_IM_FMT_I, G_IM_SIZ_8b, 64, 32, 0, G_TX_NOMIRROR | G_TX_CLAMP,
+                        G_TX_NOMIRROR | G_TX_WRAP, 6, 5, G_TX_NOLOD, G_TX_NOLOD);
+
+    gDPSetCombineLERP(POLY_XLU_DISP++, TEXEL0, PRIMITIVE, PRIM_LOD_FRAC, TEXEL0, TEXEL0, 0, PRIMITIVE, 0, PRIMITIVE,
+                      ENVIRONMENT, COMBINED, ENVIRONMENT, 0, 0, 0, COMBINED);
+    gDPSetRenderMode(POLY_XLU_DISP++, G_RM_PASS, G_RM_ZB_CLD_SURF2);
+    gSPClearGeometryMode(POLY_XLU_DISP++, G_FOG | G_LIGHTING | G_TEXTURE_GEN | G_TEXTURE_GEN_LINEAR);
+    gSPSetGeometryMode(POLY_XLU_DISP++, G_ZBUFFER | G_SHADE | G_SHADING_SMOOTH);
+    gDPPipeSync(POLY_XLU_DISP++);
+
+    gDPSetEnvColor(POLY_XLU_DISP++, this->altEnvColor.r, this->altEnvColor.g, this->altEnvColor.b, this->altEnvColor.a);
+
+    CLOSE_DISPS(gfxCtx);
+}
+
+void (*sSetupHandlers[])(GraphicsContext* gfxCtx, EffectBlure* this, Vtx* vtx) = {
+    EffectBlure_SetupSimple,
+    EffectBlure_SetupSimpleAlt,
+};
+
+s32 D_801AE268 = 0; // unused
```

Comment:
```suggestion
static s32 D_801AE268 = 0; // unused
```

Reply:
I didn't include the `static` for these unnamed symbols so they can remain in the map file and wouldn't conflict with anything, but I can add `static` if it's preferred

Reply:
I always assumed static prevented conflicts with other files by keeping it local, but that could very well be a misunderstanding on my part. I'm not sure what the standard is in OoT for unused variables set to zero, but could you just call it `sIsZero` and be done with it?